commands/addPreviewLinkChannel.js

```diff
@@ -0,0 +1,27 @@
+const DiscordUtil = require('../common/discordutil.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const ClassDB = require('../database/class-db')
+const { ChannelType } = require('discord-api-types/v9');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('addpreviewlinkchannel')
+		.setDescription('Add a channel to use the preview link feature')
+        .setDefaultPermission(false)
+        .addChannelOption(option =>
+            option.setName('channel')
+                .setDescription('The channel')
+                .addChannelTypes([ChannelType.GuildText, ChannelType.GuildPublicThread, ChannelType.GuildPrivateThread])
+                .setRequired(true)),
+	async execute(interaction) {
+        const options = interaction.options
+        const channelID = options.getChannel('channel').id;
+
+        await interaction.deferReply()
+
+        var addedChannelCorrectly = DiscordUtil.addPreviewChannel(channelID, interaction)
```

Comment:
you can totally ignore this, but should we use `let` instead of `var` as we add or make updates to be more inline with latest js standards?